mingw-w64-git/git-wrapper.c

```diff
@@ -120,8 +120,13 @@ static void setup_environment(LPWSTR top_level_path, int full_path)
 	int len;
 
 	/* Set MSYSTEM */
+	#if defined(IS_ARM64) && IS_ARM64 == true
+	swprintf(msystem, sizeof(msystem),
```

Comment:
This works correctly:

![image](https://user-images.githubusercontent.com/17739158/103210796-03115b80-4907-11eb-8f46-dbb182142b98.png)